RemoteSpiNNaker/RemoteSpiNNakerWeb/src/main/resources/log4j.properties

```diff
@@ -22,6 +22,8 @@ log4j.appender.console.Target=System.err
 log4j.appender.console.layout=org.apache.log4j.PatternLayout
 log4j.appender.console.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
 
+#log4j.logger.uk.ac.manchester.cs.spinnaker.rest.utils.ErrorCaptureResponseFilter=TRACE, console
```

Comment:
Are these commented out lines necessary?

Reply:
Just something I found useful, but you don't want it on in normal operation.  Nice to keep so I don't have to remember how to do this next time!